tests/kit/utils/cloud.ts

```diff
@@ -176,8 +181,25 @@ export async function enableCloudWorkspace(page: Page) {
   await waitForEditorLoad(page);
   await clickNewPageButton(page);
 }
+
 export async function enableCloudWorkspaceFromShareButton(page: Page) {
-  await page.getByTestId('local-share-menu-button').click();
+  const shareMenuButton = page.getByTestId('local-share-menu-button');
+  expect(await shareMenuButton.isVisible()).toBeTruthy();
+  const maxAttempts = 5;
+  let attempt = 0;
+  let menuVisible = false;
+
+  while (!menuVisible && attempt < maxAttempts) {
+    try {
+      await shareMenuButton.click();
+      menuVisible = await page.getByTestId('local-share-menu').isVisible();
```

Comment:
why not use page.waitForSelector to check element exists ?

Reply:
As I'm not quite sure whether the instability of this test is due to the need to wait for the appearance of the modal or the button being unresponsive, I am unable to reproduce this locally.

Reply:
try use waitForSelector ?